srcpkgs/python3-aiosignal/template

```diff
@@ -0,0 +1,16 @@
+# Template file for 'python3-aiosignal'
+pkgname=python3-aiosignal
+version=1.3.1
+revision=1
+build_style=python3-module
+hostmakedepends="python3-setuptools python3-wheel"
+depends="python3-frozenlist"
+checkdepends="python3-pytest python3-pytest-asyncio python3-pytest-cov
+ python3-frozenlist"
+short_desc="List of registered asynchronous callbacks"
+maintainer="Arjan Mossel <arjanmossel@gmail.com>"
+license="Apache-2.0"
+homepage="https://github.com/aio-libs/aiosignal"
+distfiles="${PYPI_SITE}/a/aiosignal/aiosignal-${version}.tar.gz"
+checksum=54cd96e15e1649b75d6c87526a6ff0b6c1b0dd3459f43d9ca11d48c339b68cfc
+make_check=no # frozenlist ModuleNotFoundError
```

Comment:
The tests run fine for me, so there must be something on your end that needs to be resolved. In any case, please reinstate the tests.